00-basics/002-javascript/freeCodeCamp_03_18.js

```diff
@@ -0,0 +1,331 @@
+function welcomeToBooleans() {
+    return true;
+
+}
+console.log(welcomeToBooleans());
+
+function  trueOrFalse(wasThatTrue) {
+    if (wasThatTrue) {
+        return "Yes, that true";
+    }
+    return "No, that was false";
```

Comment:
Yes, this is the best way to do branching between two options.

Common mistake is to structure it like this:
```
    if (wasThatTrue) {
        return "Yes, that true";
    } else {
    return "No, that was false";
    }
```

but that else is completely redundant here, if first condition is true, we are doing early return and will never reach the rest of the code anyway.